worldedit-bukkit/src/main/java/com/sk89q/worldedit/bukkit/BukkitBlockCommandSender.java

```diff
@@ -53,7 +53,7 @@ public BukkitBlockCommandSender(WorldEditPlugin plugin, BlockCommandSender sende
         this.plugin = plugin;
         this.sender = sender;
         this.location = BukkitAdapter.adapt(sender.getBlock().getLocation());
-        this.uuid = new UUID(location.toVector().toBlockPoint().hashCode(), location.getExtent().hashCode());
```

Comment:
I know we were doing it before, but I would prefer if we used a standard UUID generation scheme, such as UUIDv4 (Random) seeded with the name, or UUIDv5 (namespaced).

Reply:
Or perhaps, since it's built-in to Java, UUIDv3 (also namespaced): https://docs.oracle.com/javase/8/docs/api/java/util/UUID.html#nameUUIDFromBytes-byte:A-